src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -534,6 +614,27 @@ public enum MidpointRounding
         AwayFromZero = 1,
         ToEven = 0,
     }
+    public sealed class OperatingSystem
+    {
+        private OperatingSystem() { }
+        public OperatingSystem(System.PlatformID platform, System.Version version) { }
+        public System.PlatformID Platform { get { return default(System.PlatformID); } }
+        public string ServicePack { get { return default(string); } }
+        public System.Version Version { get { return default(System.Version); } }
+        public object Clone() { return default(object); }
+        public override string ToString() { return default(string); }
+        public string VersionString { get { return default(string); } }
+    }
+    public enum PlatformID
+    {
+        Win32S = 0,
```

Comment:
Could we add some comments to these platform id's. I have no clue what Win32S is. Is that Nano?